tracing-log/src/lib.rs

```diff
@@ -1,6 +1,25 @@
 //! Adapters for connecting unstructured log records from the `log` crate into
 //! the `tracing` ecosystem.
 //!
+//! ## Convert log records to tracing `Event`s
+//!
+//! To make logs seen as tracing events, set up `LogTracer` as logger by calling
```

Comment:
How about
```suggestion
//! To convert [`log::Record`]s as [`tracing::Event`]s, set `LogTracer` as the default logger by calling
```
(and, add links to `log::Record` and `tracing::Event` below)